### ISSUE TITLE
Save Game Sync

### ISSUE THREAD

**Issue by sergen213** [Heroic-Games-Launcher/HeroicGamesLauncher#1340]
Hello,

I'll just cut to the chase, auto sync or downloading save games from cloud never worked for me. Log says local save is new and doesn't download it from the cloud also when I enabled auto sync it said the same thing and just erased my cloud save by uploading local one. I just lost my Borderlands 3 and Control save :'( and now I can't even share the logs with you because it shows cloud saves as it is uploaded today. You may add a prompt to use either cloud or local so we won't lose our save games in future.

Thank you

**Comment by flavioislima** [comment]
The options to download and upload, that are used with the auto-sync, don't overwrite the saves.
So if your cloud saves is newer, it won't overwrite it, same for local save.
The Force Upload and Force Download on the other Hand will overwrite it, since will force the latest.

Personally, I have never had issues with heroic overwriting my saves because I check if the folder is correct and make a backup of my saves to another folder from time to time.
On the other hand, I lost saves with Steam several times, recently with Death stranding and twice with Sekiro.

The bottom line is: never trust your saves to any software, make your own backups 👍🏽
